### PR TITLE
update german locale

### PR DESCRIPTION
**Pull request by Busfreak**

### CODE REVIEW

File: app/Locale/de_DE/translations.php [hunk 1]
```diff
@@ -1184,44 +1184,44 @@
...
+    '%s removed a subtask for the task %s' => '%s hat eine Teilaufgabe in der Aufgabe %s entfernt',
+    'Comment removed' => 'Kommentar entfernt',
+    'Subtask removed' => 'Teilaufgabe entfernt',
+    '%s set a new internal link for the task #%d' => '%s hat eine neue interne Verbindung in der Aufgabe #%d erstellt',
+    '%s removed an internal link for the task #%d' => '%s hat eine interne Verbindung von der Aufgabe #%d entfernt',
+    'A new internal link for the task #%d have been defined' => 'Eine neue interne Verbindung für die Aufgabe #%d wurde definiert',
+    'Internal link removed for the task #%d' => 'Interne Verbindung in der Aufgabe #%d wurde entfernt',
+    '%s set a new internal link for the task %s' => '%s hat eine neue interne Verbindung in der Aufgabe %s erstellt',
+    '%s removed an internal link for the task %s' => '%s hat eine interne Verbindung von der Aufgabe %s entfernt',
+    'Automatically set the due date on task creation' => 'Ablaufdatum automatisch bei Erstellung einer Aufgabe setzen',
```
Comment (stinnux) [inline app/Locale/de_DE/translations.php]: I would call it "Fälligkeitsdatum" instead of Ablaufdatum.

File: app/Locale/de_DE/translations.php [hunk 2]
```diff
@@ -1184,44 +1184,44 @@
...
+    'Move the task to another column when closed' => 'Aufgabe in eine andere Spalte verschieben, wenn diese geschlossen wird',
+    'Move the task to another column when not moved during a given period' => 'Aufgabe in eine andere Spalte verschieben, wenn diese in einer bestimmten Zeit nicht verschoben wurde',
+    'Dashboard for %s' => 'Dashboard für %s',
```
Comment (stinnux) [inline app/Locale/de_DE/translations.php]: You translate Task and Subtask, but not dashboard. hmm. in my opinion you should either translate all of them or leave all common words such as Task and Subtask in english.

File: app/Locale/de_DE/translations.php [hunk 3]
```diff
@@ -1184,44 +1184,44 @@
...
+    'Activity stream for %s' => 'Aktivitätenstrom für %s',
+    'Calendar for %s' => 'Kalender für %s',
+    'Notifications for %s' => 'Benachrichtigungen für %s',
+    'Subtasks export' => 'Teilaufgaben Export',
+    'Tasks exportation' => 'Aufgaben Export',
+    'Assign a color when the task is moved to a specific swimlane' => 'Einer Aufgabe eine Farbe zuweisen, wenn diese in eine bestimmte Swimlane verschoben wird',
+    'Assign a priority when the task is moved to a specific swimlane' => 'Einer Aufgabe eine Priorität zuweisen, wenn diese in eine bestimmte Swimlane verschoben wird',
+    'User unlocked successfully.' => 'Benutzer erfolgreich entsperrt.',
+    'Unable to unlock the user.' => 'Benutzer kann nicht entsperrt werden.',
+    'Move a task to another swimlane' => 'Aufgabe in eine andere Swimlane verschieben',
```
Comment (stinnux) [inline app/Locale/de_DE/translations.php]: Same here. Either translate all or leave all. but it's just my opinion.

File: app/Locale/de_DE/translations.php [hunk 4]
```diff
@@ -1184,44 +1184,44 @@
...
+    'Creator Name' => 'Name des Erstellers',
+    'Time spent and estimated' => 'Aufgewendete und erwartete Zeit',
```
Comment (stinnux) [inline app/Locale/de_DE/translations.php]: i would rather call it "geschätzte Zeit" instead of "erwartete Zeit".

**Comment by stinnux** [comment]
I am not the developer and have no votes here, so don't misinterpret my notes. it is just my opinion, since i am a german guy ;)

File: app/Locale/de_DE/translations.php (hunk 1 above)
Comment (Busfreak) [inline app/Locale/de_DE/translations.php]: Ablaufdatum has always been Ablaufdatum in the german translation, so I didn't want to change it now.

File: app/Locale/de_DE/translations.php (hunk 2 above)
Comment (Busfreak) [inline app/Locale/de_DE/translations.php]: Dashboard has always been dashboard in the german translation, so I didn't want to change it now.
What would you prefer?

File: app/Locale/de_DE/translations.php (hunk 3 above)
Comment (Busfreak) [inline app/Locale/de_DE/translations.php]: What would you prefer? Schwimmbahn?

File: app/Locale/de_DE/translations.php (hunk 4 above)
Comment (Busfreak) [inline app/Locale/de_DE/translations.php]: see above

**Comment by Busfreak** [comment]
There has been a lot of time since the last update of german locales, why didn't you create a PR the way you like? But you are free to correct the spelling :smile:

**Pull request merged**